docs/mesos/DCOS.md

```diff
@@ -0,0 +1,134 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.18.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Installing Calico in DCOS
+This guide provides instructions for installing Calico and its
+prerequisites on a Mesos DCOS cluster.
+
+##Prerequisites
+
+### Mesos Master and Agents
+
+You will need to be running a cluster with at least one Mesos master
+and and at least one Mesos agent (though we recommend having two or
+more agents).
+
+We currently support Mesos version `v0.28.0` and on for use with DCOS.
+Additionally, we support Centos7 as an operating system for use with
+the Unified Containerizer and Docker Containerizer, though additional
+OS's *may* work (though have not been fully tested!) with the Docker
+Containerizer.
+
+### Etcd
+Calico uses etcd as its datastore.
```

Comment:
That's a short paragraph.  I think it's worth covering the following:
-  Calico requires an etcd datastore.
-  A service discovery URL is required to access the datastore - which can be configured in the Calico service advanced options.
-  You can quickly spin up an etcd cluster using the Universe package as follows.  By default, the Calico service is configured to use the default service discovery URL used by this etcd service (in other words - if you install etcd as below, you don't need to edit the service discovery url in the Calico config).